.github/workflows/ci.yml

```
@@ -0,0 +1,64 @@
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+
+      - name: build binary
+        run: |
+          cd src
+          cargo build --release
+          ../target/release/camino-conduit --version
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)